airflow/ui/src/pages/Providers.tsx

```diff
@@ -0,0 +1,75 @@
+/*!
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+import { Box, Heading, Link, Table } from "@chakra-ui/react";
+
+import { useProviderServiceGetProviders } from "openapi/queries";
+
+const embedLinks = (text: string) => {
+  const urlRegex = /http(s)?:\/\/[\w.-]+(\.?:[\w.-]+)*([/?#][\w\-._~:/?#[\]@!$&'()*+,;=.%]*)?/gu;
```

Comment:
```suggestion
  const urlRegex = /http(s)?:\/\/[\w.-]+(\.?:[\w.-]+)*([/?#][\w\-._~:/?#[\]@!$&'()*+,;=.%]*)?/g;
```

I think it should be consistent with how we detect urls in the rest of the airflow application. If unicode matching is more appropriate we can update all the regexp in a follow up PR.

Reply:
What should I do if removing `u` flag fails the static checks?

Reply:
Indeed, https://github.com/apache/airflow/pull/45535#discussion_r1933945704